examples/gallery/symbols/text_symbols.py

```diff
@@ -10,10 +10,10 @@
 different characters) and *string* can be a letter or a text string
 (less than 256 characters). Optionally, you can append **+f**\ *font* to
 select a particular font [Default is :gmt-term:`FONT_ANNOT_PRIMARY`] as well as
-**+j**\ *justify* to change the justification [Default is CM]. Outline
-and fill color of the text symbols can be customized via the ``pen``
-and ``color`` parameters, respectively.
-
+**+j**\ *justify* to change the justification [Default is CM]. Outline color of
+the text symbols can also be defined via the ``style`` parameter [Default is
+black] while the fill color and the outline width can be customized via the
+``color`` and ``pen`` parameters, respectively.
```

Comment:
```suggestion
black], the fill color can be set with the ``color`` parameter, and the 
outline width can be customized with the ``pen`` parameter.
```
To minimize confusion for non-native English speakers (and maybe some native English speakers), I think it makes it easier to follow along to have the parameter name next to what is being set.

Reply:
Thanks @willschlitzer .